source/usage-examples/find-operations.txt

```diff
@@ -2,7 +2,8 @@
 Find Operations
 ===============
 
-.. default-domain:: mongodb
+.. meta::
+   :description: See examples of querying and retrieving data from MongoDB by using the {+driver-long+}.
```

Comment:
C: since "querying" and "retrieving data from" aren't exactly parallel, could we just remove one of them?

Reply:
I think they are two separate actions, so maybe "querying data and retrieving data" would be appropriate.
Querying is the action of formulating instructions and sending the command to performing matching whereas retrieving is accessing the results, perhaps from a cursor.